From 9bead5592c7906c5b9b3bfe50349a273479fd42b Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Thu, 26 Nov 2015 09:49:10 +0800
Subject: [PATCH] staging: ion: add to compile rockchip ion driver

Add to compile rockchip ion driver to support rockchip ion

Change-Id: I3cb4f5a2110d6e0ec073eb6c48ee08796110188e
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 drivers/staging/android/ion/Kconfig  | 6 ++++++
 drivers/staging/android/ion/Makefile | 1 +
 2 files changed, 7 insertions(+)

diff --git a/drivers/staging/android/ion/Kconfig b/drivers/staging/android/ion/Kconfig
index 345234624492..d35e49baba34 100644
--- a/drivers/staging/android/ion/Kconfig
+++ b/drivers/staging/android/ion/Kconfig
@@ -27,6 +27,12 @@ config ION_DUMMY
 	  one doesn't have access to hardware drivers that
 	  use ION.
 
+config ION_ROCKCHIP
+	tristate "Ion for Rockchip"
+	depends on ARCH_ROCKCHIP && ION
+	help
+	  Choose this option if you wish to use ion on an Rockchip.
+
 config ION_TEGRA
 	tristate "Ion for Tegra"
 	depends on ARCH_TEGRA && ION
diff --git a/drivers/staging/android/ion/Makefile b/drivers/staging/android/ion/Makefile
index b56fd2bf2b4f..df1c16912674 100644
--- a/drivers/staging/android/ion/Makefile
+++ b/drivers/staging/android/ion/Makefile
@@ -6,5 +6,6 @@ obj-$(CONFIG_ION) += compat_ion.o
 endif
 
 obj-$(CONFIG_ION_DUMMY) += ion_dummy_driver.o
+obj-$(CONFIG_ION_ROCKCHIP) += rockchip/
 obj-$(CONFIG_ION_TEGRA) += tegra/
 
-- 
2.35.3

